your-first-cluster.md

```diff
@@ -0,0 +1,512 @@
+This guide covers setting up and using Cluster API using Docker infrastructure, - which is the same way CAPI runs its end-to-end tests. To set up a Cluster using AWS, Azure GCP and many more see the [CAPI quick-start guide](https://cluster-api.sigs.k8s.io/user/quick-start.html)
+
+- [Linux](#Linux)
+    - [The management cluster](#the-management-cluster)
+    - [Your first workload cluster](#your-first-workload-cluster)
+- [macOS](#MacOS)
+    - [The management cluster](#the-management-cluster-1)
+    - [Your first workload cluster](#your-first-workload-cluster-1)
+- [Windows](#Windows)
+    - [The management cluster](#the-management-cluster-2)
+    - [Your first workload cluster](#your-first-workload-cluster-2)
+
+
+# Linux 
+
+## The Management Cluster
+
+A Cluster API Management Cluster is a Kubernetes Cluster where the components that make up the CAPI control plane are installed. These components include the Custom Resources used to create and manage a cluster and the controllers that operate on them.
+
+The first step in creating a management cluster is to create a Kubernetes cluster to host the CAPI components.
+
+```bash
+sh ./scripts/create-kind-cluster.sh
+```
+
+Once the cluster has completed provisioning you should be able to check it's healthy using:
+
+```bash
+kubectl get nodes
+```
+
+In this case we're using the Docker infrastructure provider - so we need a the Docker provider (CAPD), the Core Cluster API provider (CAPI), the Kubeadm Bootstrap provider (CAPBK) and the Kubeadm Control Plane provider to be installed on the cluster. In addition we need Cert Manager to be installed on the system.
+
+
+Cluster API's CLI - `clusterctl` is used to install the CAPI Management Cluster components - luckily it's able to handle installing all of the above as well as the Custom Resource Definitions used by the Cluster API controllers.
+
+```bash
+export CLUSTER_TOPOLOGY=true
+clusterctl init --infrastructure docker:v1.2.2
```

Comment:
To use the offline flow we have to set the --config arg (see: https://hackmd.io/@sbueringer/kubecon-na-22-capi-lab)